### ISSUE TITLE
randomEngineStateProducer product not found in 140.0 and 150.0

### ISSUE THREAD

**Issue by makortel** [cms-sw/cmssw#30095]
In the latest IB CMSSW_11_2_X_2020-06-02-2300 the workflows 140.0 and 150.0 started to fail (they did not fail in an earlier IB so the failures are likely independent of https://github.com/cms-sw/cmssw/issues/30074). The failure is
```
An exception of category 'ProductNotFound' occurred while
   [0] Calling EventProcessor::runToCompletion (which does almost everything after beginJob and before endJob)
Exception Message:
The RandomNumberGeneratorService is trying to restore
the state of the random engines by reading a product from
the LuminosityBlock with input tag "InputTag:  label = randomEngineStateProducer, instance = beginLumi, process = @skipCurrentProcess".
It could not find the product.
Either the product in the LuminosityBlock was dropped or
not produced or the configured input tag is incorrect or there is a bug somewhere
```
https://cmssdt.cern.ch/SDT/cgi-bin/logreader/slc7_amd64_gcc820/CMSSW_11_2_X_2020-06-02-2300/pyRelValMatrixLogs/run/150.0_HydjetQ_B12_5020GeV_2018+HydjetQ_B12_5020GeV_2018+DIGIHI2018+RECOHI2018+HARVESTHI2018/step3_HydjetQ_B12_5020GeV_2018+HydjetQ_B12_5020GeV_2018+DIGIHI2018+RECOHI2018+HARVESTHI2018.log#/

**Comment by makortel** [comment]
From the PRs merged in CMSSW_11_2_X_2020-06-02-2300 the https://github.com/cms-sw/cmssw/pull/29990 looks like a plausible candidate causing the failures by modifying HI event content.

**Comment by makortel** [comment]
assign operations

**Comment by perrotta** [comment]
Indeed I think the issue actually comes from #29990 , as you suspected.

```IOMCRAW``` is included in the ```FEVTSIMEventContent```; but then when:
```PYTHON
FEVTSIMEventContent.outputCommands.extend(FEVTEventContent.outputCommands)
```
then the "drop *" statement at the beginning of ```FEVTEventContent``` now resets it

I try to prepare a fix

**Comment by perrotta** [comment]
assign reconstruction

**Comment by perrotta** [comment]
A fix is available in #30097

**Comment by makortel** [comment]
Thanks @perrotta!

**Comment by perrotta** [comment]
+1
- Fixed with #30097
